common/rpc/rpc_common_test.go

```diff
@@ -0,0 +1,133 @@
+// The MIT License
```

Comment:
You've added few files and they are following different naming convention. I guess it is ok, as soon as we have mess in our file names anyway. But let's use snake_case.go moving forward. This is what I see in `go` itself and famous projects like `docker`.